### ISSUE TITLE
Equivalent of cellProps for column headers?

### ISSUE THREAD

**Issue by Susan123456789** [imballinst/react-bs-datatable#138]
Is there an equivalent to cellProps for column headers?  Like headerProps?  I would like to add a class name to a header (specifically a max-width for the column).  Thanks!

**Comment by imballinst** [comment]
@Susan123456789 I have published `3.4.0-alpha.0` for testing, result can be seen in this sandbox: https://codesandbox.io/s/react-bs-datatable-3-4-0-alpha-0-thprops-zl8qi9?file=/src/App.js. Does this suffice what you need? Sample usage is as follows:

```js
// App.js
const STORY_HEADERS = [
  {
    prop: "name",
    title: "Name",
    isFilterable: true
  },
  {
    prop: "username",
    title: "Username"
  },
  {
    prop: "location",
    title: "Location"
  },
  {
    prop: "date",
    title: "Last Update"
  },
  {
    prop: "score",
    title: "Score",
    isSortable: true,
    thProps: {
      className: "th--score"
    }
  }
];
```

The class:

```css

.table > thead > tr > .th--score {
  background-color: rgb(255, 0, 0, 0.5);
}
```

**Comment by imballinst** [comment]
I am closing this issue -- `3.4.0` is published: https://www.npmjs.com/package/react-bs-datatable. Please feel free to re-open if the newest version doesn't solve this case.
